### DISCUSSION TITLE
Nulleinspeisung mit zwei Inverter geht nicht

### DISCUSSION THREAD

**Discussion by tthnn** [tbnobody/OpenDTU#1785] [General]
Hi. 
Habe einen hm 400 und einen hm 350 an der Open DTu on battery .
Beim dynamic Power Limiter steuert er aber immer nur einen Inverter an und schaltet den zweiten aus .
Wo ist dort der Fehler ? Muss noch etwas zusätzlich eingestellt werden ?

**Comment by MetaChuh** [comment]
@tthnn 

du bist im falschen github projekt.

bitte schließe dieses discussion hier, und öffne eine neue dort: [`OpenDTU-OnBattery/discussions`](https://github.com/helgeerbe/OpenDTU-OnBattery/discussions)

thx
